scripts/tasks_retrival/Genecorpus_tasks_creation.py

```diff
@@ -236,26 +129,20 @@ def main(
 ):
     for task_name, task_file in DATA_FILE_NAMES.items():
         input_path_or_url = verify_source_of_data(
-            input_file, allow_downloads=allow_downloads
+            input_file, url=DATA_URL, allow_downloads=allow_downloads
         )
         if allow_downloads:
             full_path = input_path_or_url + task_file
         else:
             full_path = Path(input_path_or_url) / task_file
```

Comment:
why do you need this if statement? 
lets do: `Path(input_path_or_url) / task_file` in both cases.

Reply:
Yes I need it because in the case of the URL the "/" will yield an error because it's not a path object.